core/primitives/src/stateless_validation.rs

```diff
@@ -19,13 +19,85 @@ use near_primitives_core::types::{AccountId, Balance, BlockHeight, ShardId};
 /// This is a messy workaround until we know what to do with NEP 483.
 type SignatureDifferentiator = String;
 
-// TODO(shreyan): Fill this struct
 #[derive(Debug, Clone, PartialEq, Eq, BorshSerialize, BorshDeserialize)]
-pub struct PartialEncodedStateWitness {}
+pub struct PartialEncodedStateWitness {
```

Comment:
can you add a brief comment about what this is, what this represents. Even it is clear now it may be helpful for future readers of the code.